types/react-relay/test/react-relay-tests.tsx

```diff
@@ -717,7 +717,7 @@ requestSubscription(
 ReactRelayContext.Consumer.prototype;
 ReactRelayContext.Provider.prototype;
 
-const MyRelayContextProvider: React.FunctionComponent = ({children}) => {
+const MyRelayContextProvider: React.FunctionComponent<{ children?: React.ReactNode }> = ({children}) => {
```

Comment:
Can't find consensus in https://github.com/DefinitelyTyped/DefinitelyTyped/pull/59143

Reply:
ya we cannot push forward with this without community consultation firstly